common/modules/gcp-project/main.tf

```diff
@@ -72,121 +72,145 @@ variable "apis_solely_for_audit_configuration" {
   ]
 }
 
+# The following two data resources define the iam policy for the projects
+# managed by the CI and the projects managed by developers. The member field
+# accepts an empty list but I wasn't able to find a way to interpolate in
```

Comment:
So this means we can create and assign a policy with roles with empty `members`? If so I think we can merge this into just one policy. I'll add a code stub how I think it can work...